controller/api-controllers.js

```diff
@@ -70,14 +71,31 @@ exports.postComment = (req, res, next) => {
 
 exports.patchArticles = (req, res, next) => {
   const { article_id } = req.params;
-  const promises = [
-    updateArticle(req.body, article_id),
-    checkExists("articles", "article_id", article_id),
-  ];
+
+  const promises = [updateArticleVotes(req.body, article_id)];
   Promise.all(promises)
     .then((data) => {
       const article = data[0];
+      if (article === undefined) {
+        return Promise.reject({ status: 404, msg: "Not found" });
+      }
```

Comment:
We can now do some short hands to make this neater. for example `if(!article)` You don't have to change this now, or even at all if you don't want to